core/frontend/src/ViewState.ts

```diff
@@ -63,6 +63,49 @@ export interface ModelDisplayTransformProvider {
   getModelDisplayTransform(modelId: Id64String, baseTransform: Transform): Transform;
 }
 
+/** Interface adopted by caller that wants to set up a perspective or orthographic view for a [[ViewState3d]].
+ * @see [[ViewState3d.LookAtPerspectiveOrOrtho]].
+ * @beta
+ */
+export interface LookAtArgs {
+  /** The new location of the camera/eye. */
+  readonly eyePoint: XYAndZ;
+  /** A vector that orients the camera's "up" (view y). This vector must not be parallel to the view direction. */
+  readonly upVector: Vector3d;
+  /** The distance from the eyePoint to the front plane. If undefined, the existing front distance is used. */
+  readonly frontDistance?: number;
+  /** The distance from the eyePoint to the back plane. If undefined, the existing back distance is used. */
+  readonly backDistance?: number;
+  /** Used for providing onExtentsError. */
+  readonly opts?: ViewChangeOptions;
+}
+
+/** Interface adopted by caller that wants to set up a perspective view for a [[ViewState3d]].
+ * @see [[LookAtArgs]]
+ * @see [[ViewState3d.lookAtPerspectiveOrOrtho]].
+ * @beta
+ */
+export interface LookAtPerspectiveArgs extends LookAtArgs {
+  /** The direction in which the view should look. */
+  readonly targetPoint: XYAndZ;
+  /** The new size (width and height) of the view rectangle. The view rectangle is on the focus plane centered on the targetPoint.
+   * If newExtents is undefined, the existing size is unchanged.
+   */
+  readonly newExtents?: XAndY;
+}
+
+/** Interface adopted by caller that wants to set up an ortho view for a [[ViewState3d]].
+ * @see [[LookAtArgs]]

```

Comment:
Don't need to `@see` base types - the documentation generator automatically produces links to them.
`Arguments to [[ViewState3d.lookAtPerspectiveOrOrtho]].`

Reply:
removed the @see for base types.